tidy3d/plugins/design/parameter.py

```diff
@@ -30,7 +30,14 @@ class Parameter(Tidy3dBaseModel, ABC):
     def _values_unique(cls, val):
         """Supplied unique values."""
         if (val is not None) and (len(set(val)) != len(val)):
-            raise pd.ValidationError("Supplied 'values' were not unique.")
+            raise pd.ValidationError(
+                model=Parameter,
+                errors=[
+                    pd.error_wrappers.ErrorWrapper(
+                        ValueError("Supplied 'values' were not unique."), "/Parameter"
```

Comment:
what is this? haven't seen this ErrorWrapper before

Reply:
in fact, it should be fine to just `raise ValueError(("Supplied 'values' were not unique.")` here and below. I think we phased out using `pd.ValidationError` in validators.

Reply:
Pd was failing all the existing `pd.ValidationError` calls because it expects these error wrappers. It's a bit unnecessary here for how we're using it so I'll replace them all with `ValueError` instead